package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0-alpha.1",
   "description": "",
   "main": "index.js",
+  "repository": {
```

Comment:
Oh no, I meant you can get rid of the object entirely and the value is just a string :) "facebook/fbjs" is understood to be git by default.